### DISCUSSION TITLE
Can`t style disabled button hover

### DISCUSSION THREAD

**Discussion by DavidManookyan** [chakra-ui/chakra-ui#3515] [General]
<Button
            _disabled={{
                background: 'black',
                hover: {
                    background: 'green',
                }
            }}
            bg="red.100"
            disabled={!link}
            onClick={signUp}
            primary={+true}>
            Sign Up
        </Button>

**Comment by malashevskyi** [comment]
It works in this way:
```javascript
<Button
  _disabled={{
    background: 'black',
  }}
  sx={{ '&:disabled:hover': { background: 'green' } }}
  bg="red.100"
  disabled={true}
>
  Sign Up
</Button>
```
